cypress/e2e/functional/tile_tests/tile_navigator_spec.js

```diff
@@ -78,45 +101,62 @@ context("Tile Navigator", () => {
     cy.get(".tile-navigator .zoom-level").should("have.text", "166%");
     clueCanvas.clickToolbarButton("drawing", "zoom-out");
     cy.get(".tile-navigator .zoom-level").should("have.text", "160%");
+    clueCanvas.deleteTile("drawing");
+
+    clueCanvas.addTile("geometry");
+    cy.get(".tile-navigator .zoom-level").should("have.text", "100%");
+    clueCanvas.clickToolbarButton("geometry", "zoom-in");
+    cy.get(".tile-navigator .zoom-level").should("have.text", "125%");
+    clueCanvas.clickToolbarButton("geometry", "zoom-in");
+    cy.get(".tile-navigator .zoom-level").should("have.text", "156%");
+    clueCanvas.clickToolbarButton("geometry", "zoom-out");
+    cy.get(".tile-navigator .zoom-level").should("have.text", "125%");
+    clueCanvas.clickToolbarButton("geometry", "zoom-out");
+    cy.get(".tile-navigator .zoom-level").should("have.text", "100%");
+    clueCanvas.clickToolbarButton("geometry", "zoom-out");
+    cy.get(".tile-navigator .zoom-level").should("have.text", "80%");
+    clueCanvas.deleteTile("geometry");
   });
+
   it("is at the bottom of the drawing tile by default but can be moved to the top", () => {
```

Comment:
I guess this should be changed to simply "bottom of the tile" since it checks both tile types?